tartiflette/language/validators/query/executable_definitions.py

```diff
@@ -0,0 +1,35 @@
+from tartiflette.language.ast.base import ExecutableDefinitionNode
+from tartiflette.utils.errors import graphql_error_from_nodes
+
+
+class ExecutableDefinition:
+
+    RULE_NAME = "executable-definitions"
+
+    def __init__(self):
+        self._extensions = {
+            "rule": "5.1.1",
+            "spec": "June 2018",
+            "details": "https://graphql.github.io/graphql-spec/June2018/#sec-Executable-Definitions",
+            "tag": self.RULE_NAME,
+        }
+
+    def validate(self, target, path, **__):
+        bad_nodes = []
+        errors = []
+
+        for definition in target:
+            if not isinstance(definition, ExecutableDefinitionNode):
+                bad_nodes.append(definition)
+
+        if bad_nodes:
+            errors.append(
```

Comment:
I think we should have a dedicated error for each not executable definition node instead of having one with each nodes location. What do you think?

This could simplify the whole `validate` method:
```python
return [
    graphql_error_from_nodes(
        message="This definition is not executable.",
        nodes=definition,
        extensions=self._extensions,
    )
    for definition in target
    if not isinstance(definition, ExecutableDefinitionNode)
]
```

Reply:
Yep, i'll refactor this one to work like the other "duplicate" rules

Reply:
Hum, sorry I mis-read this, I think an only error with multiple location is a better here. We have the same things with the "uniqueness" errors. Same error at multiple location.